src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -229,6 +231,38 @@ public DocumentConventions()
             }
         }
 
+        /// <summary>
+        /// Set the timeout for the second broadcast attempt.
+        /// Default: 30 Seconds.
+        ///
+        /// Upon failure of the first attempt the request executor will resend the command to all nodes simultaneously. 
+        /// </summary>
+        public TimeSpan? SecondBroadcastAttemptTimeout
```

Comment:
Set defaults in `DocumentConventions` ctor and remove nullable

Reply:
deon